src/main/java/com/quizapp/enterprise/persistence/GameTracker.java

```diff
@@ -96,19 +104,21 @@ public void updateGameState(GameStatus gameStatus, String gameId){
     }
 
     public Player getPlayer(String gameCode, String username){
-        return getGameByCode(gameCode).getPlayers().stream().filter(p -> p.getPlayerUsername().equals(username)).findFirst().get();
+        return getGameByCode(gameCode)
+                .getPlayers()
+                .stream()
+                .filter(p -> p.getPlayerUsername()
+                        .equals(username))
+                .findFirst()
+                .get();
```

Comment:
reformatted the streams to use indentation so the code may be cleaner and easier to read.
This recommended by the "Iterators and Streams" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
Nice use of streams, by the development group.